Makefile

```diff
@@ -115,7 +115,7 @@ pb: vendor-build generate-pb fmt
 RACE_OPT := 
 DEBUG_OPT := 
 CGO_DEBUG_OPT :=
-CGO_OPTS=CGO_CFLAGS="-I$(ROOT_DIR)/cgo" CGO_LDFLAGS="-L$(ROOT_DIR)/cgo -lmo"
+CGO_OPTS=CGO_CFLAGS="-I$(ROOT_DIR)/cgo" CGO_LDFLAGS="-L$(ROOT_DIR)/cgo -lmo -lm"
```

Comment:
You need to find out all the -lmo and update it.   I think somewhere in our code, we still have cgo instruction in go file.

Need to update everything